src/components/color-picker/react-color/ColorPicker.tsx

```diff
@@ -1,7 +1,7 @@
-import lodashDebounce from 'lodash/debounce.js';
 import type { CSSProperties } from 'react';
 import { useCallback, useEffect, useRef, useState } from 'react';
 
+import { debounce as internalDebounce } from '../../utils/index.js';
```

Comment:
There's no reason to call this "internal". We don't do that with other internal functions.